File: JJB/1126/옷가게_할인_받기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int price) 
{
    int answer = 0;
    double discount = 0;
    
    if(price >= 500000)
        discount = 0.20;
    else if(price >= 300000)
        discount = 0.10;
    else if(price >= 100000)
        discount = 0.05;
    
    answer = static_cast<int>(price * (1-discount));
    
    return answer;
}

File: JJB/1126/옷가게_할인_받기.py
def solution(price):
    answer = 0
    discount = 0
    
    if price >= 500000:
        discount = 0.20
    elif price >= 300000:
        discount = 0.10
    elif price >= 100000:
        discount = 0.05
        
    answer = int(price * (1-discount))
    
    return answer